www/src/components/layout/Sidebar.js

```diff
@@ -158,6 +161,8 @@ ref
 
   return wrapLink(wrapTooltip(
     <Flex
+      onMouseEnter={() => setHovered(true)}
```

Comment:
onMouseEnter can sometimes be an unreliable event, I think you can actually use the `_hover` attribute to do this, right @dherault ?

Reply:
_hover works only for css-related styling. Might work here.